central/plugins/org.jboss.tools.central/src/org/jboss/tools/central/messages.properties

```diff
@@ -42,3 +42,13 @@ or provide newer versions of software when performing an update or install.\n\
 I understand these conditions, and still want to enable Early Access features.
 SoftwarePage_nothingToInstall_title=Everything up-to-date
 SoftwarePage_nothingToInstall_description=All selected connectors are already installed and up-to-date.
+
+noEngineError_message=Webkit is not installed. To use JBoss Tools Central please install libwebkitgtk library with your favourite package manager.To install it execute the following command:\n\
```

Comment:
it depends on GTK version. Under GTK3 yes, under GTK2 no. But installing webkit is easier. I'd rather ask user to install package then download xulrunner and add flag to ini file